### DISCUSSION TITLE
Translathon

### DISCUSSION THREAD

**Discussion by zwiastunsw** [irdpl/wcag#10] [Pomysły]
**Pomysł**: Gdybyśmy tak, z okazji różnych, na wzór wydarzeń typu _hackathon_, organizowali wydarzenia nazwane np. ***translathon***. 
**Cel**: (Prze)tłumaczenie dokumentów WCAG - objaśnień, technik , a z czasem innych dokumentów W3C. 

**Najbliższa okazja**: 12 kwietnia - rocznica opublikowania KRI, 13 kwietnia - rocznica publikacji oficjalnej polskiego tłumaczenia WCAG. Pierwszy _translathon_ mógłby trwać np. od 12 do 16 kwietnia. 
**Zadanie**: Zadaniem pierwszego _translathonu_ mogłoby być przetłumaczenie objaśnień KS.

Organizujemy? Jeśli tak: potrzebny  jest każdy pomysły  na to, jak to dobrze zrobić.

**Comment by speplinski** [comment]
Myślę, że warto byłoby zaprosić osoby, które zajmują się UX Writing'iem, czy pisaniem "po prostu", albo podesłać informacje do agencji, które takie osoby mają.

**Comment by rotnicki** [comment]
Pomysł translatonu popieram.
Trzeba by pomyśleć nad prostymi zasadami i zdecydować co wystawiamy do tłumaczenia. To powinny być jakieś jednolite "kawałki". Czyli np. understanding do jednego konkretnego kryterium sukcesu.
Można też spróbować uderzyć z komunikatem na uczelnie i dotrzeć do studentów anglistyki. Może dr Izabela Mrochen by tu mogła coś podpowiedzieć?
